packages/base/core/src/less/variables/components/popover.less

```
@@ -30,7 +30,7 @@
 @oui-popover-close-button-icon-size: rem-calc(16);
 @oui-popover-close-button-icon-color: @p-500;
 @oui-popover-close-button-icon-color_hover: @p-700;
-@oui-popover-close-button-outline-offset: calc(-@oui-a11y-outline-offset - @oui-a11y-outline-width - @oui-collapsible-border-width);
+@oui-popover-close-button-outline-offset: calc(-@oui-a11y-outline-offset - @oui-a11y-outline-width - @oui-popover-border-width);
```

Jus t
```suggestion
@oui-popover-close-button-outline-offset: calc(?@oui-a11y-outline-offset - @oui-a11y-outline-width - @oui-popover-border-width);
```
this is intentional right ?

---

It's intentional, it's a negative sum. We want the outline to be inside the component